modules/codegen/src/main/scala/com/twilio/guardrail/generators/Endpoints.scala

```diff
@@ -11,7 +11,18 @@ import ScalaGenerator._
 import EndpointsGenerator._
 
 object Endpoints extends (CodegenApplication[ScalaLanguage, ?] ~> Target) {
-  val interpDefinitionPM: DefinitionPM[ScalaLanguage, ?] ~> Target       = ProtocolSupportTermInterp or ModelProtocolTermInterp
+  import com.twilio.guardrail.protocol.terms.protocol.{ PackageObjectImports, ProtocolSupportTerm }
+  val java8timeCirceInterp = new (ProtocolSupportTerm[ScalaLanguage, ?] ~> Target) {
+    import scala.meta._
+    def apply[A](value: ProtocolSupportTerm[ScalaLanguage, A]): Target[A] = value match {
+      case PackageObjectImports() =>
+        ProtocolSupportTermInterp(value).map { values =>
+          values :+ q"import io.circe.java8.time._"
```

Comment:
Please tell me if I am missing something, but doesn't this just add back the lines I removed in [CirceProtocolGenerator.scala](https://github.com/twilio/guardrail/pull/363/files#diff-ef22233a73a3c72d40555514fad41afbL370)? That would break on Scala 2.13/Circe 0.12 again.

My suggested change is to remove the imports by default, since they are only needed on Scala 2.11, and add a clarification to the documentation for 2.11 users.

Reply:
It seems as though there's no workaround for scalajs currently, until the next release. For that reason, I added them back in, effectively locking scalajs projects to 2.12, but permitting Scala projects to move on to 2.13. An alternate strategy would be to convince the example runner to inject the import for the scalajs target; I went for this option to limit impact, as it'll get reverted on the next Circe release, I think.

Does this impact you? If so, feel free to force push to take a different approach with your branch, I was focused on striking a balance during this Circe migration.